doc/Makefile.am

```diff
@@ -1,2 +1,2 @@
-dist_man_MANS = cdrwtool.1 mkfs.udf.8 mkudffs.8 pktsetup.8 wrudf.8
+dist_man_MANS = cdrwtool.1 mkfs.udf.8 mkudffs.8 pktsetup.8 wrudf.8 udffsck.8
```

Comment:
You need to specify also fsck.utf.8

Reply:
Oh, ok. 

Is there possibility to make is as symlink? I don't want to keep two same files.

Reply:
Oh, I forgot I already made that file.

Added to list here:  2ee62c0